projects/lkft-android/master.jinja2

```diff
@@ -0,0 +1,69 @@
+{% extends "testcases/android/master.jinja2" %}
+
+{% set timeout_common_tests_after_boot = timeout_common_tests_after_boot|default(20) %}
+
+{% set lava_job_name = "lkft-android-" + KERNEL_BRANCH + '-' + BUILD_NUMBER + '-' + job_name %}
+{% set LAVA_JOB_PRIORITY = TEST_LAVA_JOB_PRIORITY %}
+
+{########## LAVA JOB DEFINITION ##############}
+{% block visibility %}
+  group:
+    - {{TEST_LAVA_JOB_GROUP}}
+{% endblock visibility %}
+
+{% block metadata %}
+  android.build: "{{BUILD_NUMBER}}"
+  android.name: "{{JOB_NAME}}"
+  android.url: "{{REFERENCE_BUILD_URL}}"
+  android.version: "{{ANDROID_VERSION}}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
+  android.build.vendor.fingerprint: "{{ANDROID_VENDOR_FINGERPRINT}}"
+{% endif %}
+{% if ANDROID_GSI_FINGERPRINT is defined %}
+  android.build.gsi.url: "{{ANDROID_GSI_URL}}"
+  android.build.gsi.fingerprint: "{{ANDROID_GSI_FINGERPRINT}}"
+{% endif %}
+{% if TUXSUITE_DOWNLOAD_URL is defined %}
+  tuxsuite.download.url: "{{TUXSUITE_DOWNLOAD_URL}}"
+{% endif %}
+  lkft.build.config: "{{LKFT_BUILD_CONFIG}}"
+  git branch: "{{KERNEL_BRANCH}}"
+  git repo: "{{KERNEL_REPO}}"
+  git commit: "{{KERNEL_COMMIT}}"
+  git describe: "{{KERNEL_DESCRIBE}}"
+  build-url: "{{BUILD_URL}}"
+  toolchain: "{{TEST_METADATA_TOOLCHAIN}}"
+  series: lkft
+{% block job_specific_metadata %}
+{% endblock job_specific_metadata %}
```

Comment:
This can be dropped. Please use  '{{ super() }}'

Reply:
yes, it could be, but instead of 
```
{% block metadata %}
{{ super() }}
...
{% endblock %}
```
we only need to write
```
{% block job_specific_metadata %}
...
{% endblock job_specific_metadata %}
```
so that we don't need to remember to write the '{{ super() }}' line, and job_specific_metadata block makes it more clear on what the lines in it are.

Reply:
I guess you understand but I mean like this: http://ix.io/3U06

I think its clear enough since you specify metadata in a test specific file...

Reply:
here is my reason to use job_specific_metadata `so that we don't need to remember to write the '{{ super() }}' line, and job_specific_metadata block makes it more clear on what the lines in it are.` 
what's advantage there with the "{{super}}" method?